.github/workflows/queasars_deployment.yml

```diff
@@ -36,13 +36,16 @@ jobs:
     name: Check Version Consistency
     runs-on: ubuntu-latest
     steps:
-      - if: ${{ inputs.version != '' }}
+      - if: ${{ github.event_name == 'workflow_dispatch' }}
         name: Check workflow version input
         shell: bash
+        env:
+          VERSION_INPUT: ${{ inputs.version }}
         run: |
-          if [[ $inputs.version != @(v[0-9].[0-9].[0-9]) ]]; then exit 1; fi
-          echo "VERSION=${inputs.version:1}" >> $GITHUB_ENV
-      - if: ${{ inputs.version == '' }}
+          echo $VERSION_INPUT
```

Comment:
Is this a debug statement? If so, you can probably remove it.

Reply:
Correct, this was a debugging statement. I removed it know. Thanks for finding it!